chains/container-chains/nodes/frontier/src/command.rs

```diff
@@ -60,6 +54,19 @@ fn load_spec(id: &str, para_id: ParaId) -> std::result::Result<Box<dyn ChainSpec
         "dev" => Box::new(chain_spec::development_config(para_id, vec![])),
         "template-rococo" => Box::new(chain_spec::local_testnet_config(para_id, vec![])),
         "" | "local" => Box::new(chain_spec::local_testnet_config(para_id, vec![])),
+
+        // dummy container chain spec, it will not be used to actually spawn a chain
+        "container-chain-unknown" => Box::new(
```

Comment:
I believe this is needed because of how we create the runner later:

```rust
let runner = cli.create_runner(&cmd.base.container_run.normalize())?;

runner.run_node_until_exit(|config| async move {
```

So the dummy spec could be avoided if we rewrite a bit the runner creation code, but I tried and it's not easy.

So it's fine